server/database/stampRepository.ts

```diff
@@ -762,24 +794,34 @@ export class StampRepository {
         {}
       );
 
-      const summarized = Object.keys(grouped).map(key => 
+      const summarized = Object.keys(grouped).map((key) =>
         summarize_issuances(grouped[key])
       );
-
       // Use the passed xcpBalances for quantity info
       return summarized.map((summary: StampBalance) => {
-        const xcp_balance = xcpBalances.filter(
-          balance => balance.cpid === summary.cpid
-        ).reduce((acc,balance)=>acc+balance.quantity,0);
-        const utxos = xcpBalances.filter(balance => balance.cpid === summary.cpid && balance.utxo_address).map(balance => ({ quantity: balance.quantity, utxo_address: balance.utxo_address}))
-        const unbound_quantity = xcpBalances.filter(balance =>  balance.cpid === summary.cpid && !balance.utxo_address).length
+        const xcp_balance = xcpBalances
+          .filter((balance) => balance.cpid === summary.cpid)
+          .reduce((acc, balance) => acc + balance.quantity, 0);
+        const utxos = xcpBalances
+          .filter(
+            (balance) => balance.cpid === summary.cpid && balance.utxo_address
+          )
+          .map((balance) => ({
+            quantity: balance.quantity,
+            utxo_address: balance.utxo_address,
+          }));
+        const unbound_quantity = xcpBalances
```

Comment:
this is the database call which shouldn't differentiate between bound and unbound here. we need to typically keep all balances the total balance with the exception of the /api/v2/stamps/balance endpoing

Reply:
I think that's not database call 
it's just using the javascript for filtering data.

Reply:
ah ok i see. this looked like it was part of getstamps

Reply:
👍